.github/workflows/conda.yml

```diff
@@ -48,36 +58,53 @@ jobs:
 
       - uses: conda-incubator/setup-miniconda@v2
         name: Vanilla install from conda
-        if: ${{ matrix.cfg.openeye == 'false' }}
         with:
           python-version: ${{ matrix.cfg.python-version }}
           activate-environment: test
           environment-file: devtools/conda-envs/conda.yaml
```

Comment:
Is there a way to use the `setup-miniconda` action just to get the base env, and then create a new env with an explicit call to `conda create ...`? 

I think this PR has good improvements, but the basic problem all along was that `setup-miniconda` was resolving package deps differently than a fresh miniconda deployment would have.